### PR TITLE
Issue template

### PR DESCRIPTION
**Pull request by peiyaoh**
### Description

- Add issue templates for bug report, feature request, and documentation update.

### Related Jira Tickets

-

### Comments 

-

Please ensure all items are complete before requesting a review:

- [ ] **Code Completed**: All code for the feature or bugfix has been implemented.
- [ ] **Unit Tests Added**: Unit tests are written for all new functionality, covering edge cases where applicable.
- [ ] **Integration Tests (where necessary)**: Integration tests are added for interactions between modules or components.
- [ ] **All Tests Passing**: All unit and integration tests are passing.
- [ ] **Type Safety Checked**: Ensure TypeScript types are correctly defined and used. Avoid `any` types without proper justification.
- [ ] **Typedoc Comments Added**: Ensure all functions, classes, interfaces, and types are documented using TypeDoc syntax.
- [ ] **Code Formatting**: All code files are formatted according to the project’s standards (e.g., Prettier, ESLint) and no linting errors remain.
- [ ] **No Console Logs**: Remove any `console.log` or other debugging output from the code.
- [ ] **Documentation Updated**: Update relevant project documentation (e.g., README, configuration guides) if necessary.

---

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Expected behavior
+
+A clear and concise description of what you expected to happen.
+
```
[the remark this answers is not in the document]
Comment (numerator) [reply .github/ISSUE_TEMPLATE/bug.md:23]: Maybe amend the text under Describe the bug to say "A clear and concise description of what the bug is, including when it occurs and what how the system behaves."

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+If applicable, add screenshots to help explain your problem.
+
+### Environment (please complete the following information)
+
+- OS: [e.g., Mac OS, Windows, Linux]
+- OS Version: [e.g., 25]
+- Node Version: [e.g., 22]
+
+### Additional context
```
[the remark this answers is not in the document]
Comment (numerator) [reply .github/ISSUE_TEMPLATE/bug.md:34]: I think it could be useful and optional. Something like
Severity
[ ] Very Low: e.g., cosmetic, grammatical
[ ] Low: e.g., annoying but still functional
[ ] Medium: e.g., things mostly work but functionality is impeded
[ ] High: e.g., needed functionality not accessible
[ ] Very High: e.g., showstopper/system crasher

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (peiyaoh) [reply .github/ISSUE_TEMPLATE/bug.md:23]: Added! Thanks for the language suggestion.

[the remark this answers is not in the document]
Comment (jadrubio) [reply .github/ISSUE_TEMPLATE/bug.md:23]: I think we are srtill missing the actual behaviors.  typically in bug reports you put the issue, the steps to replicate, what is supposed to happy and what actually happens

[the remark this answers is not in the document]
Comment (peiyaoh) [reply .github/ISSUE_TEMPLATE/bug.md:23]: Thanks! I just added the "actual behavior" section. I was originally unsure whether it was redundant, given that we already ask them to describe the bug in the first section. I can see that it would be beneficial to have a dedicated section to ask for more details, in case reporters left a concise description of the bug. The parallel of expected vs. actual behavior will also make it easier for the developers who take on the issue.

**Review by jadrubio — APPROVED** [review]

**Review by numerator — APPROVED** [review]
